static/app/components/modals/demoEndModal.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {fetchGuides} from 'sentry/actionCreators/guides';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import {navigateTo} from 'sentry/actionCreators/navigation';
+import {Client} from 'sentry/api';
+import Button from 'sentry/components/button';
+import ModalTask from 'sentry/components/onboardingWizard/modalTask';
+import {SidebarPanelKey} from 'sentry/components/sidebar/types';
+import {IconClose} from 'sentry/icons/iconClose';
+import {t, tct} from 'sentry/locale';
+import SidebarPanelStore from 'sentry/stores/sidebarPanelStore';
+import {useRouteContext} from 'sentry/utils/useRouteContext';
+
+type Props = ModalRenderProps & {orgSlug: string | null; tour: string};
+
+function DemoEndingModal({tour, closeModal, orgSlug}: Props) {
+  const api = new Client();
+
+  const routeContext = useRouteContext();
+  const {router} = routeContext;
+
+  function togglePanel(panel: SidebarPanelKey) {
+    SidebarPanelStore.togglePanel(panel);
+  }
+
+  let cardTitle, body, guides, location;
+  switch (tour) {
+    case 'issues':
+      cardTitle = 'Issues Tour';
+      body = 'completing the Issues tour';
```

Comment:
I would have each of these actually be the entire string

```
'Thank you for completing the Issues tour. Learn about other Sentry features by starting another tour.',
````

````
'Thank you for checking out the different tabs. Learn about other Sentry features by starting another tour.',
```

The problem with injecting these substrings like his is that the translators will have a harder time translating these strings without context.

You can imagine

imagine a language where the grammar is different than english,

`completing the Performance tour`

and 

`Thank you for [body]. Learn about other Sentry features by starting another tour.`

the translator sees those two strings completely separately. In a different language words might change based on whats inside of `[body]`.

Reply:
Agree with @evanpurkhiser ! Not sure if it helps, but for the same reason, I always prefer to use `tct()` over `t()` as it provides more context to the person who is translating

Reply:
ohhh , that's a good point , thanks for letting me know